backend/src/main/java/quarano/reference/SymptomDto.java

```diff
@@ -23,4 +26,5 @@
 
 	private @Alphabetic	String name;
 	private boolean isCharacteristic;
+	private Map<Language, String> translations;
```

Comment:
We have a service for extracting all symptoms. I though it would help us to check "from outside" if we have the translations for all of them or not. Without looking into DB. What do you think?

Reply:
Yes, that could be useful. But shouldn't it be better separated from the client interface?